api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -113,6 +116,201 @@ message RateLimitPerRoute {
     IGNORE = 2;
   }
 
+  // Global rate limiting :ref:`architecture overview <arch_overview_global_rate_limit>`.
+  message RateLimitSpecifier {
```

Comment:
Actually, you can still call it RateLimit, it's in a separate package. Then you'd have no need to create separate v4alpha/rate_limit